### PR TITLE
Gender pronouns and other neutral terms

### PR DESCRIPTION
**Pull request by kingthorin**
Closes OWASP/www-community#229

- Replace "she" with they
- Replace "he" with they
- Replace "him" with them
- Replace "him/her" with them
- Replace "her" with their
- Replace "his" with their
- Replace "he/she" with they
- Replace "he or she" with they
- Replace "blacklist" and variants with deny list
- Replace "whitelist" and variants with allow list

Didn't touch any content under social/chapters all day.

### CODE REVIEW

File: pages/OWASP_Application_Security_FAQ.md [hunk 1]
```diff
@@ -860,7 +860,7 @@ not tampered.
 
 ## Can I trust the IP address of a user I see in my audit logs? Could a user be spoofing/impersonating their IP address?
 
-A bad guy who wants to hide his actual IP address might use a service
+A bad person who wants to hide theiractual IP address might use a service
```
Comment (robhanlon22) [inline pages/OWASP_Application_Security_FAQ.md:863]:
```suggestion
A bad person who wants to hide their actual IP address might use a service
```

File: pages/attacks/DOM_Based_XSS.md [hunk 2]
```diff
@@ -126,10 +126,10 @@ flaw, but if not all users have downloaded the patch then those users
 are still vulnerable to this type of attack.
 
 Ivan Ristic did some research and proposed some server side defenses
-against this type of attack in his presentation "Protecting Web
+against this type of attack in the presentation "Protecting Web
 Applications from Universal PDF XSS: A discussion of how weird the web
 application security world has become" at the [2007 OWASP Europe AppSec
-Conference](OWASP_AppSec_Europe_2007_-_Italy "wikilink") in Milan. His
+Conference](OWASP_AppSec_Europe_2007_-_Italy "wikilink") in Milan, the
```
Comment (robhanlon22) [inline pages/attacks/DOM_Based_XSS.md:132]:
```suggestion
Conference](OWASP_AppSec_Europe_2007_-_Italy "wikilink") in Milan. The
```

File: pages/attacks/Forced_browsing.md [hunk 3]
```diff
@@ -39,14 +39,14 @@ Enumeration, Directory Enumeration, and Resource Enumeration.
...
-resources by modifying URL parameters. The user1 wants to check his
+resources by modifying URL parameters. The user1 wants to check their
 on-line agenda through the following URL:
 
 ` www.site-example.com/users/calendar.php/user1/20070715 `
 
 In the URL, it is possible to identify the username (âuser1â) and
 the date (mm/dd/yyyy). If the user attempts to make a forced browsing
-attack, he could guess another userâs agenda by predicting user
+attack, they could guess another user's agenda by predicting user
```
Comment (robhanlon22) [inline pages/attacks/Forced_browsing.md:49]:
```suggestion
attack, they could guess another user's agenda by predicting user
```

**Comment by kingthorin** [comment]
Thanks @robhanlon22

**Pull request merged**